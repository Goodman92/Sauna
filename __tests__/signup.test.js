import React from 'react';
import renderer from 'react-test-renderer';

import SignupScreen from '../src/screens/signup/signup';


it('renders screen', () => {
  const tree = renderer.create(<SignupScreen/>).toJSON();

  expect(tree).toMatchSnapshot();
});



