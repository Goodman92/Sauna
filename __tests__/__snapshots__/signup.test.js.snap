// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders screen 1`] = `
<View
  style={
    Object {
      "height": "100%",
      "width": "100%",
    }
  }
>
  <Image
    source={
      Object {
        "process": [Function],
      }
    }
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {
          "height": "100%",
          "width": "100%",
        },
        undefined,
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "flex": 3,
          "justifyContent": "center",
        },
      ]
    }
  >
    <View
      style={
        Array [
          Object {
            "flexDirection": "column",
            "width": 125,
          },
        ]
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#fff",
              "textAlign": "center",
            },
            Object {
              "fontSize": 24,
              "letterSpacing": 0,
            },
          ]
        }
      >
        SIGNUP
      </Text>
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "justifyContent": "center",
            },
          ]
        }
      >
        <View
          style={
            Array [
              Object {
                "borderColor": "#fff",
                "borderLeftWidth": 0,
                "borderRadius": 1,
                "borderRightWidth": 0,
                "borderWidth": 0.5,
                "flex": 3,
                "height": 3,
              },
            ]
          }
        />
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#fff",
                "flex": 2,
                "textAlign": "center",
              },
            ]
          }
        >
          to
        </Text>
        <View
          style={
            Array [
              Object {
                "borderColor": "#fff",
                "borderLeftWidth": 0,
                "borderRadius": 1,
                "borderRightWidth": 0,
                "borderWidth": 0.5,
                "flex": 3,
                "height": 3,
              },
            ]
          }
        />
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#fff",
              "textAlign": "center",
            },
            Object {
              "fontSize": 34,
              "letterSpacing": -0.5,
            },
          ]
        }
      >
        SAUNA
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "flex": 6,
        "paddingLeft": 50,
        "paddingRight": 50,
      }
    }
  >
    <View
      onResponderRelease={[Function]}
      onStartShouldSetResponder={[Function]}
      pointerEvents="auto"
    >
      <View
        style={
          Object {
            "backgroundColor": "transparent",
            "borderBottomColor": "rgba(255, 255, 255, 1)",
            "borderBottomWidth": 0.5,
            "height": 64,
            "paddingBottom": 8,
            "paddingTop": 32,
          }
        }
      >
        <View
          style={
            Object {
              "position": "absolute",
              "top": 36,
            }
          }
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            numberOfLines={1}
            style={
              Object {
                "color": "rgba(255, 255, 255, 1)",
                "fontSize": 16,
              }
            }
          >
            Username
          </Text>
        </View>
        <View
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <TextInput
            allowFontScaling={true}
            autoCapitalize="sentences"
            disableFullscreenUI={true}
            editable={true}
            error={null}
            onBlur={[Function]}
            onChange={[Function]}
            onChangeText={[Function]}
            onContentSizeChange={[Function]}
            onFocus={[Function]}
            selectionColor="#fff"
            style={
              Array [
                Object {
                  "flex": 1,
                  "margin": 0,
                  "padding": 0,
                  "top": 2,
                },
                Object {
                  "color": "rgba(0, 0, 0, .87)",
                  "fontSize": 16,
                  "height": 24,
                  "textAlign": "left",
                },
                undefined,
              ]
            }
            textContentType="username"
            underlineColorAndroid="transparent"
            value=""
          />
        </View>
      </View>
      <View
        style={
          Object {
            "flexDirection": "row",
            "height": 8,
          }
        }
      >
        <View
          style={
            Object {
              "flex": 1,
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "flex-start",
                "bottom": 0,
                "left": 0,
                "paddingVertical": 4,
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              numberOfLines={1}
              style={
                Object {
                  "backgroundColor": "transparent",
                  "color": "rgb(213, 0, 0)",
                  "fontSize": 0,
                  "opacity": 0,
                }
              }
            />
          </View>
          <View
            style={
              Object {
                "alignItems": "flex-start",
                "bottom": 0,
                "left": 0,
                "paddingVertical": 4,
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              numberOfLines={1}
              style={
                Object {
                  "backgroundColor": "transparent",
                  "color": "#fff",
                  "fontSize": 12,
                  "opacity": 1,
                }
              }
            />
          </View>
        </View>
      </View>
    </View>
    <View
      onResponderRelease={[Function]}
      onStartShouldSetResponder={[Function]}
      pointerEvents="auto"
    >
      <View
        style={
          Object {
            "backgroundColor": "transparent",
            "borderBottomColor": "rgba(255, 255, 255, 1)",
            "borderBottomWidth": 0.5,
            "height": 64,
            "paddingBottom": 8,
            "paddingTop": 32,
          }
        }
      >
        <View
          style={
            Object {
              "position": "absolute",
              "top": 36,
            }
          }
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            numberOfLines={1}
            style={
              Object {
                "color": "rgba(255, 255, 255, 1)",
                "fontSize": 16,
              }
            }
          >
            Email
          </Text>
        </View>
        <View
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <TextInput
            allowFontScaling={true}
            autoCapitalize="sentences"
            disableFullscreenUI={true}
            editable={true}
            error={null}
            onBlur={[Function]}
            onChange={[Function]}
            onChangeText={[Function]}
            onContentSizeChange={[Function]}
            onFocus={[Function]}
            selectionColor="#fff"
            style={
              Array [
                Object {
                  "flex": 1,
                  "margin": 0,
                  "padding": 0,
                  "top": 2,
                },
                Object {
                  "color": "rgba(0, 0, 0, .87)",
                  "fontSize": 16,
                  "height": 24,
                  "textAlign": "left",
                },
                undefined,
              ]
            }
            textContentType="emailAddress"
            underlineColorAndroid="transparent"
            value=""
          />
        </View>
      </View>
      <View
        style={
          Object {
            "flexDirection": "row",
            "height": 8,
          }
        }
      >
        <View
          style={
            Object {
              "flex": 1,
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "flex-start",
                "bottom": 0,
                "left": 0,
                "paddingVertical": 4,
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              numberOfLines={1}
              style={
                Object {
                  "backgroundColor": "transparent",
                  "color": "rgb(213, 0, 0)",
                  "fontSize": 0,
                  "opacity": 0,
                }
              }
            />
          </View>
          <View
            style={
              Object {
                "alignItems": "flex-start",
                "bottom": 0,
                "left": 0,
                "paddingVertical": 4,
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              numberOfLines={1}
              style={
                Object {
                  "backgroundColor": "transparent",
                  "color": "#fff",
                  "fontSize": 12,
                  "opacity": 1,
                }
              }
            />
          </View>
        </View>
      </View>
    </View>
    <View
      onResponderRelease={[Function]}
      onStartShouldSetResponder={[Function]}
      pointerEvents="auto"
    >
      <View
        style={
          Object {
            "backgroundColor": "transparent",
            "borderBottomColor": "rgba(255, 255, 255, 1)",
            "borderBottomWidth": 0.5,
            "height": 64,
            "paddingBottom": 8,
            "paddingTop": 32,
          }
        }
      >
        <View
          style={
            Object {
              "position": "absolute",
              "top": 36,
            }
          }
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            numberOfLines={1}
            style={
              Object {
                "color": "rgba(255, 255, 255, 1)",
                "fontSize": 16,
              }
            }
          >
            Password
          </Text>
        </View>
        <View
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <TextInput
            allowFontScaling={true}
            autoCapitalize="sentences"
            disableFullscreenUI={true}
            editable={true}
            error={null}
            onBlur={[Function]}
            onChange={[Function]}
            onChangeText={[Function]}
            onContentSizeChange={[Function]}
            onFocus={[Function]}
            selectionColor="#fff"
            style={
              Array [
                Object {
                  "flex": 1,
                  "margin": 0,
                  "padding": 0,
                  "top": 2,
                },
                Object {
                  "color": "rgba(0, 0, 0, .87)",
                  "fontSize": 16,
                  "height": 24,
                  "textAlign": "left",
                },
                undefined,
              ]
            }
            textContentType="password"
            underlineColorAndroid="transparent"
            value=""
          />
        </View>
      </View>
      <View
        style={
          Object {
            "flexDirection": "row",
            "height": 8,
          }
        }
      >
        <View
          style={
            Object {
              "flex": 1,
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "flex-start",
                "bottom": 0,
                "left": 0,
                "paddingVertical": 4,
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              numberOfLines={1}
              style={
                Object {
                  "backgroundColor": "transparent",
                  "color": "rgb(213, 0, 0)",
                  "fontSize": 0,
                  "opacity": 0,
                }
              }
            />
          </View>
          <View
            style={
              Object {
                "alignItems": "flex-start",
                "bottom": 0,
                "left": 0,
                "paddingVertical": 4,
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              numberOfLines={1}
              style={
                Object {
                  "backgroundColor": "transparent",
                  "color": "#fff",
                  "fontSize": 12,
                  "opacity": 1,
                }
              }
            />
          </View>
        </View>
      </View>
    </View>
    <View
      onResponderRelease={[Function]}
      onStartShouldSetResponder={[Function]}
      pointerEvents="auto"
    >
      <View
        style={
          Object {
            "backgroundColor": "transparent",
            "borderBottomColor": "rgba(255, 255, 255, 1)",
            "borderBottomWidth": 0.5,
            "height": 64,
            "paddingBottom": 8,
            "paddingTop": 32,
          }
        }
      >
        <View
          style={
            Object {
              "position": "absolute",
              "top": 36,
            }
          }
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            numberOfLines={1}
            style={
              Object {
                "color": "rgba(255, 255, 255, 1)",
                "fontSize": 16,
              }
            }
          >
            Confirm password
          </Text>
        </View>
        <View
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <TextInput
            allowFontScaling={true}
            autoCapitalize="sentences"
            disableFullscreenUI={true}
            editable={true}
            error={null}
            onBlur={[Function]}
            onChange={[Function]}
            onChangeText={[Function]}
            onContentSizeChange={[Function]}
            onFocus={[Function]}
            selectionColor="#fff"
            style={
              Array [
                Object {
                  "flex": 1,
                  "margin": 0,
                  "padding": 0,
                  "top": 2,
                },
                Object {
                  "color": "rgba(0, 0, 0, .87)",
                  "fontSize": 16,
                  "height": 24,
                  "textAlign": "left",
                },
                undefined,
              ]
            }
            textContentType="password"
            underlineColorAndroid="transparent"
            value=""
          />
        </View>
      </View>
      <View
        style={
          Object {
            "flexDirection": "row",
            "height": 8,
          }
        }
      >
        <View
          style={
            Object {
              "flex": 1,
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "flex-start",
                "bottom": 0,
                "left": 0,
                "paddingVertical": 4,
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              numberOfLines={1}
              style={
                Object {
                  "backgroundColor": "transparent",
                  "color": "rgb(213, 0, 0)",
                  "fontSize": 0,
                  "opacity": 0,
                }
              }
            />
          </View>
          <View
            style={
              Object {
                "alignItems": "flex-start",
                "bottom": 0,
                "left": 0,
                "paddingVertical": 4,
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              numberOfLines={1}
              style={
                Object {
                  "backgroundColor": "transparent",
                  "color": "#fff",
                  "fontSize": 12,
                  "opacity": 1,
                }
              }
            />
          </View>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "marginTop": 20,
        }
      }
    >
      <View
        pointerEvents="box-only"
        style={
          Array [
            Object {
              "alignItems": "center",
              "borderRadius": 2,
              "flexDirection": "row",
              "height": 36,
              "justifyContent": "center",
              "paddingHorizontal": 16,
            },
            false,
            Object {
              "backgroundColor": "#fff",
              "borderColor": "rgba(0,0,0,.12)",
            },
            false,
            Object {
              "backgroundColor": "rgba(0, 0, 0, 0.12)",
            },
            Object {},
            undefined,
          ]
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#000000",
                "fontSize": 14,
                "fontWeight": "500",
              },
              false,
              undefined,
              false,
              Object {
                "color": "rgba(0, 0, 0, 0.26)",
              },
              undefined,
              undefined,
            ]
          }
        >
          SIGNUP
        </Text>
      </View>
    </View>
  </View>
</View>
`;
